src/main/java/com/epam/ta/reportportal/core/project/impl/UpdateProjectHandlerImpl.java

```diff
@@ -284,10 +286,12 @@ private void assignUser(String name, ProjectRole projectRole, List<String> assig
 		projectUser.setUser(modifyingUser);
 		projectUser.setProject(project);
 		project.getUsers().add(projectUser);
-		aclHandler.permitSharedObjects(project.getId(),
-				name,
-				ProjectRole.PROJECT_MANAGER.higherThan(projectRole) ? BasePermission.READ : BasePermission.ADMINISTRATION
-		);
+
+		List<Permission> permissions = Lists.newArrayList(BasePermission.READ);
+		if (!ProjectRole.PROJECT_MANAGER.higherThan(projectRole)) {
```

Comment:
let's make this condition more understandable using for example 'lowerThan'

Reply:
SameOrLower method has been added